rules/base.js

```diff
@@ -0,0 +1,172 @@
+module.exports = {
+  "parser": "babel-eslint",
```

Comment:
For the base rules we should remove babel as well. We don't need babel on the backend.